tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py36, py37, py38, py39, pypy3
+envlist = py37, py38, py39, py310, py311, pypy3
```

Comment:
Similar to the changes above, why do we remove `py36` from the tox? 🤔

Reply:
https://devguide.python.org/versions shows that Python 3.6 is not supported by the Python core team.  This means no more security updates like [CVE-2020-10735](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-10735).  Also, contributors could choose to start using [Python 3.7 improvements](https://docs.python.org/3.7/whatsnew/3.7.html) like `asyncio`, `data classes`, and `dicts are faster and are ordered by default`.

Looking at Python minor version on https://pypistats.org/packages/pypiserver will show you how many downloads are happening.  Supporting five versions of CPython + PyPy 3 is considered enough.  It is not recommended to keep supporting unsafe versions of CPython but if that is really needed, I can try to revert those changes.

Reply:
Hi @cclauss, thanks a lot for the good and clear explanation. This is really helpful! Now I fully understand the motivation behind this. And thanks for sharing all the relevant links. I am very much in favour of moving towards better supported and newer Python versions, but I would prefer making it in a separate step :) I would really love to add the support for newer Python features like indeed as you mention, `asyncio`, `data classes` and etc. In my opinion, this certainly will make making contributions easier and more fun. 

However, at this point, [according to the `pypistats`](https://pypistats.org/packages/pypiserver)  (thanks for referring to it 👍), Python 3.6 makes up approximately the same daily download quantity as Python 3.7, and I wouldn't like to affect the people who still rely on it without announcement, even if there are only a few. (Needless to say about Python 2, we [have approximately a quarter of all downloads](https://pypistats.org/packages/pypiserver) in Python 2). 

I would like to make the sunsetting of older versions a real thing and would be happy to cooperate on this with you if you would be interested in follow-up issues. There is already a very old discussion on #107 and I will try to revive it and identify the precise steps and announcements necessary to make this transition as smooth as possible. I hope this sounds reasonable, what do you say? So if you agree, and could add the `py36` in the matrix for this pull request, I will be very grateful 🙌